### PR TITLE
Hard Coded messages moved to language file.

### PR DESCRIPTION
**Pull request by tirjok**

### CODE REVIEW

File: app/Http/Controllers/PostController.php [hunk 1]
```diff
@@ -175,7 +175,7 @@ public function destroy($id)
     protected function checkPost($post)
     {
         if (!$post) {
-            throw new NotFoundHttpException('Post Not Found');
+            throw new NotFoundHttpException(trans('messages.post.not_found'));
```
Comment (GrahamCampbell) [inline app/Http/Controllers/PostController.php]: Exception messages must not be localized.

**Comment by GrahamCampbell** [comment]
I'm fine with changing anything that's not inside an exception.

**Comment by tirjok** [comment]
Ok, i will update my commit.

File: app/Http/Controllers/CommentController.php [hunk 2]
```diff
@@ -110,7 +110,7 @@ public function store($postId)
         ]);
 
         if (CommentRepository::validate($input, array_keys($input))->fails()) {
-            throw new BadRequestHttpException('Your comment was empty.');
+            throw new BadRequestHttpException('Comment Not Found.');
```
Comment (GrahamCampbell) [inline app/Http/Controllers/CommentController.php]: ?

**Comment by GrahamCampbell** [comment]
Thanks. :)

**Pull request merged**